### DISCUSSION TITLE
Feedback for Version 3 of the SAP Cloud SDK

### DISCUSSION THREAD

**Discussion by fwilhe** [SAP/cloud-sdk-js#3557] [General]
As announced in the [release blog post](https://blogs.sap.com/2023/03/02/sap-cloud-sdk-version-3/) we'd like to know your thoughts on Version 3 of the SAP Cloud SDK.

Please create new comments in this discussion for each feedback item.

**Comment by thulium3** [comment]
the minimum supported Node version is 18, but in Business Application Studio the Node version is still on 16.16. Are you in dialog with the BAS team  or can't you use BAS for projects containing SDK 3 then?

**Reply by marikaner** [reply]
Hey @thulium3, the SAP BAS developers are currently testing node 18 internally. So you can expect that BAS will run Node.js 18 soon. You can still switch to SAP Cloud SDK for JavaScript version 3, because:
1. This limitation is only relevant for local execution. If you're publishing to CF, e.g. the default is Node.js 18 already.
2. The SDK v3 currently does not rely on any Node.js 18 features yet. So most likely for now you can even use it locally without issues.

I hope I didn't miss anything and I hope that helps.
